frame/base/bli_offloader.c

```diff
@@ -216,42 +355,203 @@ bool bli_do_offload_gemmex_rntm_from_env
 		return false;
 	}
 
-	// figure out if the result matrix C's M*N is above or below the data type specific cutoff
-	const bool is_float_c = bli_obj_is_float ( c );
-	const bool is_compl_c = bli_obj_is_complex ( c );
-	if ( is_float_c && !is_compl_c && config->never_offload_sgemm )
-	{
-		return false;
-	}
-	else if ( !is_float_c && !is_compl_c && config->never_offload_dgemm )
-	{
-		return false;
-	}
-	else if ( is_float_c && is_compl_c && config->never_offload_cgemm )
+	if ( config->model == always ) return true;
```

Comment:
Small nitpick, these if statements have inconstant formatting with all the others. All the others I see are as follows:

if ()
{

}